packages/@dcl/inspector/src/components/Controls/Controls.css

```diff
@@ -0,0 +1,39 @@
+.component.controls {
+  padding: 3px;
+  border-radius: 1px;
+}
+
+.component.controls:hover,
+.component.controls.active {
+  background-color: #424141;
+}
+
+.component.controls > div:first-child {
+  display: flex;
+  align-items: center;
+}
+
+.component.controls > div:nth-child(2) {
```

Comment:
Wouldn't it be better to use .class to prevent this style from being broken if the markup changes?

Reply:
I come from the approach of "changing the markup should affect styles" (an old ux developer colleague directive). I honestly don't have a preference on this one (besides classes cluttering a little bit more our JSX). 
Is using classes the way to go on DCL? 
@cazala what do you think?

Reply:
mm i'm okay on using the tag name as selector when it is semantic (using label, input, button). For instance I would do this:

```tsx
<div className="MyComponent">
  <label>{label}</label>
</div>
```

```css
.MyComponent label { ... }
```

Instead of:

```tsx
<div className="MyComponent">
  <div className="label">{label}</div>
</div>
```

```css
.MyComponent .label { ... }
```

But using `div`s is not very semantic and i think it makes the code more difficult to follow than instead seeing `.menu-item` or something more meaningful, i would suggest as mendez said using classes here.